api/app/tag.py

```diff
@@ -246,11 +249,42 @@ def make_image(payload: bytes, frame_type: Literal["simple", "secure"]) -> bytes
         image_factory=StyledPilImage,
     )
 
-    # TODO: Wrap in frame using cairosvg + pillow
+    # Get the dimensions of the image in pixels
+    img_width, img_height = img.size
 
-    # Get image contents as bytes
+    # Convert the SVG frame to a PNG image
+    api_root = Path(__file__).parent.parent.absolute()
```

Comment:
This should be a constant at the module level, it will not change during the runtime of this application